src/main/java/mate/controller/car/CreateCarController.java

```diff
@@ -0,0 +1,44 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import java.util.Collections;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/car/create")
```

Comment:
![image](https://user-images.githubusercontent.com/117590814/214560799-86861343-68e1-4a33-8912-6901f4c2aaf1.png)
same for all other urls in project